TombEngine/Objects/TR2/Entity/tr2_spear_guardian.cpp

```diff
@@ -17,34 +18,102 @@ using namespace TEN::Math;
 
 namespace TEN::Entities::Creatures::TR2
 {
+	constexpr auto SPEAR_GUARDIAN_WALK_ANGLE_RATE_MAX = ANGLE(3.0f);
+	constexpr auto SPEAR_GUARDIAN_RUN_ANGLE_RATE_MAX = ANGLE(5.0f);
+
 	const auto SpearBiteLeft  = BiteInfo(Vector3(0.0f, 0.0f, 920.0f), 11);
 	const auto SpearBiteRight = BiteInfo(Vector3(0.0f, 0.0f, 920.0f), 18);
 
-	// TODO
+	constexpr auto SPEAR_GUARDIAN_SLASH_RANGE = SQUARE(SECTOR(1));
```

Comment:
Please replace `SECTOR` with `BLOCK` as we are gradually removing it. :P

Reply:
Done :)